ansible_base/activitystream/serializers.py

```diff
@@ -89,4 +93,70 @@ def _get_summary_fields(self, obj) -> dict[str, dict]:
         if related_content_obj and hasattr(related_content_obj, 'summary_fields'):
             summary_fields['related_content_object'] = related_content_obj.summary_fields()
 
+        if obj.changes is None:
+            return summary_fields
+
+        try:
+            changed_fk_fields = obj.changed_fk_fields
+        except AttributeError:
+            # The model was deleted
+            changed_fk_fields = {}
+
+        for field_name, pk in changed_fk_fields.items():
+            related_object = getattr(content_obj, field_name, None)
+            if related_object is None:
+                continue
+
+            if hasattr(related_object, 'summary_fields'):
+                summary_fields[f"changes.{field_name}"] = related_object.summary_fields()
```

Comment:
If a PATCH changes a related field (we'll say `owner` of the animal model) from Alice to Bob... then what does `related["changes.owner"]` show? Alice or Bob?

Reply:
It should show Bob in that case. I added a test just now to show that.

Reply:
After the prior steps, `owner` is then changed to Charlie. Now, the first entry when changed Alice->Bob now what does it show? Showing Charlie would be a bit weird. Just about any single choice would be a bit weird.

Reply:
Ah, okay, I see what you're saying. Good find.